src/Features/Core/Portable/FindUsages/FindUsagesHelpers.cs

```diff
@@ -96,5 +96,25 @@ private static SymbolDisplayFormat GetFormat(ISymbol definition)
         .AddParameterOptions(SymbolDisplayParameterOptions.IncludeName);
 
     public static ImmutableArray<TaggedText> GetDisplayParts(ISymbol definition)
-        => definition.ToDisplayParts(GetFormat(definition)).ToTaggedText();
+        // Range variables have to be specially handled as only ToMinimalDisplayParts can pass enough information 
+        // to the compiler to properly display the pseudo-return-type they have.
```

Comment:
Is this a bug in the compiler though? I'm having a hard time imagining why the regular symbol wouldn't have what it needs.